.github/workflows/pr-push.yml

```diff
@@ -0,0 +1,31 @@
+name: Build, Validate, and Deploy
+on:
+  pull_request: {}
+  push:
+    branches: [master]
+  workflow_dispatch:
+
+jobs:
+  main:
+    runs-on: ubuntu-20.04
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v2
```

Comment:
I am not familiar with GitHub's workflow syntax.  Where are these actions defined?  Can we include a link to their definitions, either as a comment here or in a README.md file?

Reply:
A link to https://github.com/actions/checkout/blob/main/README.md would work here.